controllers/tasks.js

```diff
@@ -142,23 +143,59 @@ const updateTaskStatus = async (req, res, next) => {
   try {
     const taskId = req.params.id;
     const { dev } = req.query;
-    const { id: userId } = req.userData;
+    const { id: userId, username } = req.userData;
     const task = await tasks.fetchSelfTask(taskId, userId);
 
     if (task.taskNotFound) return res.boom.notFound("Task doesn't exist");
     if (task.notAssignedToYou) return res.boom.forbidden("This task is not assigned to you");
-    if (task.taskData.status === "VERIFIED")
+    if (task.taskData.status === "VERIFIED" || req.body.status === "MERGED")
```

Comment:
![image](https://user-images.githubusercontent.com/57758447/194727609-4b91b45e-c920-4c92-9d5f-2e7bd3ff2a48.png)
Are all these states being checked?
like for needs review Percentage should be > 80

Reply:
for that i guess we should have a proper discussion with other members on the call